### PR TITLE
Closes #256: Write workflow integration tests for wf-ingest module

### LINKED ISSUE #256: Write workflow integration tests for wf-ingest module
There are two untested workflows in ingest module:
`eu/dnetlib/iis/wf/ingest/html/plaintext/oozie_app/workflow.xml`
`eu/dnetlib/iis/wf/ingest/webcrawl/fundings/oozie_app/workflow.xml`

### PR DESCRIPTION
**Pull request by marekhorst**

### CODE REVIEW

File: iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/html/plaintext/brokenxml/oozie_app/workflow.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+					<value>${nameNode}</value>
+				</property>
+				<property>
+					<name>queueName</name>
+					<value>${queueName}</value>
+				</property>
+				<property>
+					<name>oozieLauncherQueueName</name>
+					<value>${oozieLauncherQueueName}</value>
+				</property>
```
Comment (madryk) [inline iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/html/plaintext/brokenxml/oozie_app/workflow.xml]: Please add global section to workflow

File: iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/html/plaintext/nulltext/oozie_app/workflow.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="test-ingest_html_plaintext_nulltext">
```
Comment (madryk) [inline iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/html/plaintext/nulltext/oozie_app/workflow.xml]: Same here. `<global>` section

File: iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/html/plaintext/sampledataproducer/oozie_app/workflow.xml [hunk 3]
```diff
@@ -0,0 +1,104 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="test-ingest_html_plaintext_sampledataproducer">
+	<start to="producer" />
```
Comment (madryk) [inline iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/html/plaintext/sampledataproducer/oozie_app/workflow.xml]: global section

File: iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/webcrawl/fundings/brokenxml/oozie_app/workflow.xml [hunk 4]
```diff
@@ -0,0 +1,104 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="test-ingest_webcrawl_fundings_brokenxml">
+	<start to="producer" />
+	<action name="producer">
```
Comment (madryk) [inline iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/webcrawl/fundings/brokenxml/oozie_app/workflow.xml]: global section

File: iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/webcrawl/fundings/nulltext/oozie_app/workflow.xml [hunk 5]
```diff
@@ -0,0 +1,104 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="test-ingest_webcrawl_fundings_nulltext">
+	<start to="producer" />
```
Comment (madryk) [inline iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/webcrawl/fundings/nulltext/oozie_app/workflow.xml]: global section

File: iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/webcrawl/fundings/sampledataproducer/oozie_app/workflow.xml [hunk 6]
```diff
@@ -0,0 +1,104 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="test-ingest_webcrawl_fundings_sampledataproducer">
+	<start to="producer" />
```
Comment (madryk) [inline iis-wf/iis-wf-ingest/src/test/resources/eu/dnetlib/iis/wf/ingest/webcrawl/fundings/sampledataproducer/oozie_app/workflow.xml]: global section

**Comment by marekhorst** [comment]
Fixed by adding global section and merged.

**Pull request closed without merging**